### PR TITLE
APIG changes

### PR DESCRIPTION
**Pull request by IuliaDVSA**

### CODE REVIEW

**Review by m-coslett — APPROVED** [review]

File: src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
 import static util.DataUtil.generateRandomExcludingValues;
 
-
```
Comment (Daniel-Searle) [inline src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java:17]: Can these formatting changes be reverted

File: src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java [hunk 2]
```diff
@@ -961,6 +960,5 @@ public void testResultsTestTypesMissing() {
         testResultsSteps.statusCodeShouldBe(400);
         testResultsSteps.validatePostErrorData("testTypes", "is required");
     }
-
```
Comment (Daniel-Searle) [inline src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java:964]: Can these formatting changes be reverted

File: src/main/java/util/NoDataPathFilter.java [hunk 3]
```diff
@@ -10,15 +10,25 @@
...
     }
+
+    /**
+     * Simple method to construct the base URL with the new format following the APIG restructure
+     * @return Base url as String
+     */
+    private String returnBaseUrl() {
+        String baseUrl = loader.getNoDataBasePathUrl();
+        String branchName = loader.getBranchName();
+        if (branchName.equalsIgnoreCase("develop")) {
```
Comment (Daniel-Searle) [inline src/main/java/util/NoDataPathFilter.java:28]: This if statement could be shortened to use a ternary

Comment (IuliaDVSA) [reply src/main/java/util/NoDataPathFilter.java]: done

File: src/main/java/util/BasePathFilter.java [hunk 4]
```diff
@@ -13,18 +13,30 @@
...
     }
+
+    /**
+     * Simple method to construct the base URL with the new format following the APIG restructure
+     * @return Base url as String
+     */
+    private String returnBaseUrl() {
+        String baseUrl = loader.getBasePathUrl();
+        String branchName = loader.getBranchName();
+        if (branchName.equalsIgnoreCase("develop")) {
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/main/java/util/BasePathFilter.java]: done

File: src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java (hunk 2 above)
Comment (IuliaDVSA) [reply src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java:964]: no need for an extra blank line there

**Review by Daniel-Searle — APPROVED** [review]

**Review by m-coslett — APPROVED** [review]

**Comment by IuliaDVSA** [comment]
VTA-831 APIG changes reviewed and approved

**Pull request merged**